main.go

```diff
@@ -22,6 +22,7 @@ import (
 type Args struct {
 	Debug                   bool          `usage:"Enable debug logging"`
 	Bootstrap               string        `usage:"Specifies a file with commands to initially send to the server"`
+	StopCommand             string        `usage:"Which command to send to the server to stop it ('stop' by default)."`
```

Comment:
Please declare the default here, as:

```suggestion
	StopCommand             string        `default:"stop" usage:"Which command to send to the server to stop it ('stop' by default)."`
```

Reply:
Fixed